storagenode/storagenodedb/migrations_test.go

```diff
@@ -121,6 +125,17 @@ func TestMigrate(t *testing.T) {
 
 		// verify schema and data for each db in the expected snapshot
 		for dbName, dbSnapshot := range multiDBSnapshot.DBSnapshots {
+			// If the tables and indexes of the schema are empty, that's
+			// semantically the same as nil. Set to nil explicitly to help with
+			// comparison to snapshot.
+			schema, ok := schemas[dbName]
+			if ok && len(schema.Tables) == 0 {
```

Comment:
Tip: this can be also checked with using `go-cmp` and https://godoc.org/github.com/google/go-cmp/cmp/cmpopts#EquateEmpty option, this means you don't have to manually find all the places that need fixing.